README.md

```diff
@@ -49,3 +49,6 @@ The script allows:
  - to download the current config.txt from px4
  - to upload new config.txt to px4
  - to remove the config.txt from the px4 in order to use the original PX4 Airframe params
+
+# download_logfile.py
```

Comment:
A thought, but I'm not sure if this is a good idea:

"download log file" - from where? Docker in some VM? Flight Computer?

download_px_logfile.py or download_fc_logfile.py (FC = flight computer) might add more context to what this tool does.

I'm not sure if this is a good idea. For one, it makes the script name longer. Do what you feel best, I'm perfectly happy with the script name as-is.

Reply:
I think this is good point. The script naming should give some hint about the usage. I'll rename it to be more self explanatory